scripts/lint.bat

```diff
@@ -1,7 +1,10 @@
 @echo off
 
-dotnet format --verbosity=d --severity=warn ..\ChainSafe.Gaming.sln --exclude .\submodules
+pushd ..
```

Comment:
can this still run from repo's root directory if we use `scripts/lint.bat`

Reply:
I don't understand the question? 🤔 
The issue I had with linter is that if I run it with dotnet format ..\Chainsafe.Gaming.sln it won't run for whatever reason. Adding this pushd fixed the issue on my end (Since I always like to run linter locally).

Reply:
Ah good catch, it's not running.